### ISSUE TITLE
Documentation update request: webhook-config

### ISSUE THREAD

**Issue by hryamzik** [prometheus/alertmanager#325]
[This](https://prometheus.io/docs/alerting/configuration/#webhook-receiver-webhook-config) part of documentation is outdated, here's an example of the POST JSON:

```json
{
    "receiver": "admins-critical",
    "status": "resolved",
    "alerts": [
        {
            "status": "resolved",
            "labels": {
                "alertname": "node_down",
                "env": "prod",
                "instance": "testhost.local:9100",
                "job": "node",
                "monitor": "prometheus",
                "severity": "critical"
            },
            "annotations": {
                "description": "testhost.local:9100 of job node has been down for more than 5 minutes.",
                "summary": "Instance testhost.local:9100 down"
            },
            "startsAt": "2016-04-21T20:14:37.698Z",
            "endsAt": "2016-04-21T20:15:37.698Z",
            "generatorURL": "https://monitoring.promehteus.local/graph#%5B%7B%22expr%22%3A%22up%20%3D%3D%200%22%2C%22tab%22%3A0%7D%5D"
        }
    ],
    "groupLabels": {
        "alertname": "node_down",
        "instance": "testhost.local:9100"
    },
    "commonLabels": {
        "alertname": "node_down",
        "env": "prod",
        "instance": "testhost.local:9100",
        "job": "node",
        "monitor": "prometheus",
        "severity": "critical"
    },
    "commonAnnotations": {
        "description": "testhost.local:9100 of job node has been down for more than 5 minutes.",
        "summary": "Instance testhost.local:9100 down"
    },
    "externalURL": "https://am.promehteus.local",
    "version": "3",
    "groupKey": somenumber
}
```

**Comment by fabxc** [comment]
Closed in favor of prometheus/docs#420

**Comment by hryamzik** [comment]
Oops, I've missed there's a separate repo for this. Thanks!
